eva_submission/eload_validation.py

```diff
@@ -92,6 +94,27 @@ def _validate_sample_names(self):
             })
         self.eload_cfg.set('validation', 'sample_check', 'pass', value=not overall_differences)
 
+    def _validate_genotype_aggregation(self):
+        for analysis_alias in self.eload_cfg.query('submission', 'analyses'):
+            aggregations = [
+                detect_vcf_aggregation(vcf_file)
+                for vcf_file in self.eload_cfg.query('submission', 'analyses', analysis_alias, 'vcf_files')
+            ]
+            errors = []
+            if len(set(aggregations)) == 1 and None not in aggregations:
+                aggregation = set(aggregations).pop()
+                self.eload_cfg.set('validation', 'aggregation_check', 'analysis', str(analysis_alias), value=aggregation)
```

Comment:
```suggestion
                self.eload_cfg.set('validation', 'aggregation_check', 'analyses', str(analysis_alias), value=aggregation)
```
(sorry this probably means you have to change it in a bunch of places...)

Reply:
Done